Project.toml

```
@@ -12,13 +12,20 @@ Meshes = "eacbb407-ea5a-433e-ab97-5258b1ca43fa"
 QuadGK = "1fd47b50-473d-5c70-9696-f719f8f3bcdc"
 Unitful = "1986cc42-f94f-5a68-af5c-568840ba703d"
 
+[weakdeps]
+Enzyme = "7da242da-08ed-463a-9acd-ee780be4f1d9"
+
+[extensions]
+MeshIntegralsEnzymeExt = "Enzyme"
+
 [compat]
 CliffordNumbers = "0.1.9"
-CoordRefSystems = "0.12, 0.13, 0.14, 0.15, 0.16"
+CoordRefSystems = "0.15, 0.16"
+Enzyme = "0.13"
```

```suggestion
Enzyme = "0.13.19"
```
Downgrade still fails. Looks like we need a newer Enzyme version. I just checked locally. I think we need Enzyme v0.13.19.

---

I am not sure to be honest. From the [error message](https://github.com/JuliaGeometry/MeshIntegrals.jl/actions/runs/12319562537/job/34386898636#step:8:367 ) it looks like there was a missing method definition, but I wasn't able to find a commit where it was added in Enzyme.jl. What is also a bit strange is that locally when I tested it under Julia v1.11 I had no issues. The issue was only with Julia v1.10 (what we are also using in the Downgrade test). So the error looks a bit more involved to me and is probably not worth spending too much time to understand what exactly happenend.

---

Billy Moses is pretty helpful so we probably could get some help from him, but given the rapid pace of development on Enzyme currently, this will be old news soon anyways.

---

Weird. Agreed that it’s probably not worth spending too much energy on in the grand scheme, especially for an optional extension and given that the Julia AD ecosystem is pretty fast-moving.